### PR TITLE
edited learning objectives to be more specific and task focused

### PR DESCRIPTION
**Pull request by ejh243**
part of instructor training.

### CODE REVIEW

File: 03-data-frames.Rmd [hunk 1]
```diff
@@ -18,9 +18,11 @@ surveys <- read.csv("data/portal_data_joined.csv")
 
 > ## Learning Objectives
 >
-> * understand the concept of a `data.frame`
-> * use sequences
-> * know how to access any element of a `data.frame`
+> * describe what a `data.frame` is
+> * read data from a file into a `data.frame` and change how character strings are handled
+> * summarize the size and data types of a `data.frame` 
+> * write a command to print odd numbers from 90 to 100
```
Comment (fmichonneau) [inline 03-data-frames.Rmd]: that might be a little too specific. The goal here is for learners to be able to create sequences of their own.

File: 03-data-frames.Rmd [hunk 2]
```diff
@@ -18,9 +18,11 @@ surveys <- read.csv("data/portal_data_joined.csv")
...
+> * subset the first 3 rows and first 6 columns of a `data.frame`
```
Comment (fmichonneau) [inline 03-data-frames.Rmd]: same here, maybe replace to something more general like "subset rows or columns"

File: 05-visualization-ggplot2.Rmd [hunk 3]
```diff
@@ -25,13 +25,17 @@ structured data.
...
-> * Know to to effectively visualize data using the ggplot2 package
+> * Be able to create a ggplot object
+> * Be able to set universal plot settings
+> * Be able to modify an existing ggplot object
+> * Be able to change the aesthetics of a plot such as colour
+> * Be able to edit the axis labels 
 > * Know how to use a step-by-step approach to build complex plots
 > * Be able to create, scatter plots, box plots and time series plots
-> * Use the 'facet_wrap' and 'facet_grid' commands to create a collection of plots splitting the data by a factor variable
+> * Be able to write a command using 'facet_wrap()' and 'facet_grid()'to create a collection of plots splitting the data by a factor variable
```
Comment (fmichonneau) [inline 05-visualization-ggplot2.Rmd]: Could you please reverse this sentence: "Be able to create a collection of plots splitting the data across levels of a factor using `facet_wrap()` and `facet_grid()`.

**Comment by fmichonneau** [comment]
It looks good. Thanks!

**Comment by ejh243** [comment]
Hi,

I have made the required changes and committed them to my version and they have appeared in the pull request.

Do I need to do anything else for you to see the edits?

Eilis

**Comment by fmichonneau** [comment]
Hi Ellis,  Everything is good. Your changes are already live on the datacarpentry website. Thanks!

**Pull request merged**